### ISSUE TITLE
Reusing verification tests

### ISSUE THREAD

**Issue by kpamnany** [climate-machine/CLIMA#910]
# Description

It would be helpful to use one verification test to test different features. E.g. `test/DGmethods/compressible_Navier_Stokes/mms_bc_atmos.jl` is a superset of `test/Driver/mms3.jl`. It would be neater to have a small repository of verification tests whose behavior can be externally controlled and use them in different contexts to test different features rather than to duplicate them.

# For CLIMA Developers

- [ ] This feature can be added (if it cannot be, explain why in a comment below -- lack of technical expertise, not relevant to the scope of this project, too ambitious)
- [ ] There is a timeline for when this feature can be implemented
- [ ] The feature has been (or will be) implemented (Please link the PR)

**Comment by blallen** [comment]
Does this supersede #877? In that, we need command-line arguments in order to have this functionality, so we should have the conversation in a single location?

**Comment by kpamnany** [comment]
I don't think so. Although we do need that to do this, this is a separate issue.
